webpack/assets/javascripts/react_app/components/HostsIndex/Columns/core.js

```diff
@@ -163,7 +164,12 @@ const reportedDataColumns = [
   {
     columnName: 'ram',
     title: __('RAM'),
-    wrapper: hostDetails => hostDetails?.reported_data?.ram,
+    wrapper: hostDetails => {
+      if (!hostDetails?.reported_data?.ram) return null;
+      return NumberToHumanSize(hostDetails.reported_data.ram * 1024 * 1024, {
```

Comment:
Ideally we could extract these into a helper and re-use them in other places such as https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtVmware.js#L24, but that's not blocking.